data/net/minecraft/world/level/levelgen/structure/BuiltinStructureSets.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/world/level/levelgen/structure/BuiltinStructureSets
+	METHOD register (Ljava/lang/String;)Lnet/minecraft/resources/ResourceKey;
+		ARG 0 location
```

Comment:
```suggestion
		ARG 0 name
```
This is more analogous to the existing `register` methods which generally are using `name` or `key` (both are used interchangeably in different locations).